frontend/src/static/js/v_authorship.js

```diff
@@ -169,22 +170,29 @@ window.vAuthorship = {
       let totalBlankLineCount = 0;
 
       files.forEach((file) => {
-        const lineCnt = file.authorContributionMap[this.info.author];
-        if (lineCnt) {
-          totalLineCount += lineCnt;
-          const out = {};
-          out.path = file.path;
-          out.lineCount = lineCnt;
-          out.fileType = file.fileType;
+        // skip file not touched by author
+        if (!(this.info.author in file.authorContributionMap)) {
```

Comment:
Question: Why is this logic added for this PR?

Reply:
The original intention of `if(lineCnt)` is to check if the author has contributed to the file.
However, it will fail for binary files as I assigned line count of binary files to be 0.
So instead I check directly for the existence of the author instead.